### PR TITLE
Add 404 HTML file to top level

### PR DESCRIPTION
**Pull request by kaankabalak**

### CODE REVIEW

**Comment by kaankabalak** [comment]
The styling for this page will come when minio/gluegun/#82 is merged

**Review by harshavardhana — APPROVED** [review]

**Comment by kaankabalak** [comment]
I am currently making some changes to minio/gluegun/#82 in terms of icons, so I will be making changes to the HTML file once the changes there are merged

**Comment by kaankabalak** [comment]
I have edited the file according to the changes on minio/gluegun/#82, waiting for @rushenn's feedback there to see whether we need to make any more changes to the HTML file

**Review by rushenn — APPROVED** [review]

**Pull request merged**
